example/pubspec.yaml

```diff
@@ -13,4 +13,4 @@ dependencies:
 
 dev_dependencies:
   test: any
-  lints: ^1.0.1
+  lints: ^2.0.0
```

Comment:
Also interesting that this doesn't has any effect on us. Looks like we have some great self made rules .